functions/test/endpoints/FirestoreDataFactory.ts

```diff
@@ -332,226 +349,25 @@ export class FirestoreDataFactory{
- */
-export declare type UserOptions = {
-    first?:string, 
-    floor_id?:string,
-    house_name?:string,
-    last?:string,
-    semester_points?:number,
-    total_points?:number
-}
-
-/**
- * Type declaration for fields to add to a point log. Undefined fields will be defaulted.
- */
-export declare type PointLogOptions = {
-    id?:string,
-    approved_by?:string,
-    approved_on?:Date,
-    date_occurred?:Date,
-    date_submitted?:Date,
-    description?:string,
-    floor_id?:string,
-    point_type_id?:number,
-    point_type_name?:string,
-    point_type_description?:string,
-    rhp_notifications?:number,
-    resident_first_name?:string,
-    resident_last_name?:string,
-    resident_notifications?:number
-
-}
-
-/**
- * Type declaration for fields to add to reward. Undefined fields will be defaulted.
- */
-export declare type RewardOptions = {
-    id?: string
-    required_ppr?: number,
-    required_value?: number
-}
-
-/**
- * Type Declaration for fields to add to Link. Undefined fields will be defaulted
- */
-export declare type LinkOptions = {
-    archived?: boolean,
-    description?: string,
-    enabled?: boolean,
-    single_use?: boolean
-}
-
-export const LINK_DEFAULTS:LinkOptions = {
-    archived: false,
-    description: "Basic description",
-    enabled: true,
-    single_use: true
-}
-
-/**
- * Default fields for Reward
- */
-export const REWARD_DEFAULTS:RewardOptions = {
-    id: "Pizza Party",
-    required_ppr: 100,
-    required_value: 20000
-}
-
-/**
- * Default fields for point log
- */
-export const POINT_LOG_DEFAULTS:PointLogOptions = {
-    approved_by: "Preapproved",
-    approved_on: new Date(Date.parse("5/18/2020")),
-    date_occurred: new Date(Date.parse("5/18/2020")),
-    date_submitted: new Date(Date.parse("5/18/2020")),
-    description: "Empty Description",
-    floor_id: "4N",
-    point_type_id: 1,
-    point_type_name: "Empty Name",
-    point_type_description: "Empty Description",
-    rhp_notifications: 0,
-    resident_first_name: "TEST_FIRST",
-    resident_last_name: "TEST_LAST",
-    resident_notifications: 0
-
-}
-
-/**
- * Default fields for system preferences
- */
-export const SYSTEM_PREFERENCES_DEFAULTS:SystemPreferenceOptions = {
-    android_version: "2.0.0",
-    one_time_code: "abc",
-    competition_hidden_message: "hidden",
-    house_enabled_message: "Honors1",
-    ios_version: "1.6.2",
-    is_competition_visible: true,
-    is_house_enabled: true,
-    suggested_point_ids: "1,2,3,4",
-}
-
-/**
- * default fields for point type
- */
-export const POINT_TYPE_DEFAULTS:PointTypeOptions = {
-    description: "Empty Point Type Description",
-    name: "Empty Point Type Name",
-    is_enabled: true,
-    permission_level: 2,
-    residents_can_submit: true,
-    value: 1
-}
-
-/**
- * Default field for house
- */
-export const HOUSE_DEFAULTS:HouseOptions = {
-    color: "#5AC0C7",
-    total_points: 20,
-    num_residents: 200
-}
```

Comment:
All of the Options and the type declarations have been moved to a separate file named OptionDeclarations.ts for clarity.

CompetitionPointStatus and PointStatus are now defined in this file.